src/app/components/mno-apps/mno-app.html

```diff
@@ -109,10 +109,10 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
       <div class="col-md-12">
         <div class="well">
           <h2>{{ 'mno_enterprise.templates.dashboard.marketplace.show.price' | translate }}</h2>
-          <div translate ng-if="!vm.arePlansAvailable">
+          <div translate ng-if="!vm.orderPossible">
             {{ 'mno_enterprise.templates.dashboard.marketplace.show.no_pricing_plans_found' | translate }}
           </div>
-          <ul ng-if="vm.arePlansAvailable">
+          <ul ng-if="vm.orderPossible">
```

Comment:
Should be `ng-if="vm.pricing_plans"` so that this section is hidden if there are no plans.